lua/textcase/plugin/plugin.lua

```diff
@@ -386,6 +387,77 @@ function M.start_replacing_command()
   end
 end
 
+-- This function is like start_replacing_command but it only uses <parts_count> parts
+-- for the Subs command. For example, if the current word is "LoremIpsumDolorSit" and
+-- <parts_count> is 2 and the cursor is on the first part, then the prefille dSubs command
+-- will be ":Subs/LoremIpsum/".
+--
+-- For more examples, see the tests.
+--
+-- The keybindings to make use of this could be the following:
+-- vim.api.nvim_set_keymap("n", "gar", "<cmd>lua require('textcase').start_replacing_command_with_part(1)<CR>", {})
```

Comment:
Should the parameter be a table for extending it later?
`lua require('textcase').start_replacing_command_with_part({count = 1})<CR>`

In the future, there could be another requirement to force the start of the word
`lua require('textcase').start_replacing_command_with_part({count = 1, start = 1})<CR>` (The use case you mentioned)

Reply:
Very good point!